add rsk3 webpack demo (#44)
rsksmart/rsk3.js: demo/webpack/README.md

> ## rsk webpack demo

Reviewer: ```suggestion
## rsk3.js webpack demo
```

> 1. ```npm run build``` to generate ```./dist/main.js```
> 
> 2. open ```./src/index.html``` to view page
> 
> you can update ```./src/index.js``` code and execute build again to view page

Reviewer: ```suggestion
1. `npm run build` to generate `./dist/main.js`.
2. open `./src/index.html` to view page.

You may update the code in `./src/index.js`, and repeat the above steps.
```
